ci/compile_test.ps1

```diff
@@ -1,11 +1,4 @@
-#cmake -DCMAKE_INSTALL_PREFIX=.\inst .
-cmake .
+cmake -DCMAKE_INSTALL_PREFIX=./install .
 cmake --build . --config Release
 cmake --install . --config Release
 ctest --output-on-failure
-
-# Install on windows is complicated and not like on linux
```

Comment:
Is it possible to install a library with cmake on windows? If yes we should test it, but I haven't found well documentation on that yet.

Reply:
I seems possible indeed. cmake --install should make the work. Maybe on older CMake version things were more tricky. If I remember correctly to install it on windows you need to provide the --config * as MSVC is multi-config by default. I remember having the same problem with Ninja on Linux.

Reply:
I have found this: https://cmake.org/pipermail/cmake/2010-June/037461.html. So yeah it's possible, but I guess that you rather want to compile a dynamic library and then place it into the same folder as the .exe or just statically compile it. We should document that. Moved that into #198.